### PR TITLE
added in vGPU testing scenarios

### PR DESCRIPTION
**Pull request by noahgildersleeve**
Added in test scenarios

- Enable vGPU addon
- Select server side driver
- Try to add vGPU profiles to card that doesn't support vGPU
- Add vGPU device
- Remove device
- Add two vGPUs to vm
- add one vGPU to VM that already has active vGPU
- Try to add already used device
- Try to add too many devices in `advanced vGPU devices`

### CODE REVIEW

File: docs/content/manual/advanced/addons/2764-vgpu.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+cd cuda-samples/Samples/3_CUDA_Features/cudaTensorCoreGemm
+make
+./cudaTensorCoreGemm
+```
+
+## Add one vGPU on VM creation
+
+1. Create a VM and select the vGPU in vGPU devices
+1. After the VM is created run the pre-reqs as outlined above
+1. Run the grid installer for the vGPU driver on the VM
```
Comment (albinsun) [inline docs/content/manual/advanced/addons/2764-vgpu.md:56]: Do we need explict command or webpage reference to install GRID vGPU driver package?
e.g.
```
sudo ./NVIDIA-Linux-x86_64-version-grid.run
```

Comment (noahgildersleeve) [reply docs/content/manual/advanced/addons/2764-vgpu.md:56]: I agree that this is kind of unclear. The issue is that every different GPU will have a different URL, and you need to be an nvidia partner to log in and get the file. I don't have access to that. I'll add in a note that you need to get the KVM and grid files from Nvidia.

File: docs/content/manual/advanced/addons/2764-vgpu.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+1. Validate that the PCI devices aren't enabled
+1. Click the link to enable PCI devices
+1. Enable PCI devices in the linked addon page
+1. Wait for the status to change to `Deploy Successful`
+1. Navigate to the PCI devices page
+1. Validate that the PCI devices page is populated/populating with PCI devices
+
+# Pre-requisite Enable vGPU
+
+This can only be ran on a bare metal Harvester cluster that has an Nvidida card that support vGPU
```
[the remark this answers is not in the document]
Comment (noahgildersleeve) [reply docs/content/manual/advanced/addons/2764-vgpu.md:22]: Good catch. Thanks.

File: docs/content/manual/advanced/addons/2764-vgpu.md (hunk 1 above)
Comment (noahgildersleeve) [reply docs/content/manual/advanced/addons/2764-vgpu.md:56]: I added a link in the pre-requisites that outlines which installers/drivers you need and where to see that Nvidia requires a partner program to download them.

**Review by albinsun — APPROVED** [review]
LGTM, thank you!

**Pull request merged**